src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -50,25 +50,16 @@ public bool SupportsThrowExpression(ParseOptions options)
         public SyntaxToken ParseToken(string text)
             => SyntaxFactory.ParseToken(text);
 
-        public bool IsAwaitKeyword(SyntaxToken token)
-        {
-            return token.IsKind(SyntaxKind.AwaitKeyword);
-        }
+        public bool IsAwaitKeyword(SyntaxToken token) => token.IsKind(SyntaxKind.AwaitKeyword);
 
-        public bool IsIdentifier(SyntaxToken token)
-        {
-            return token.IsKind(SyntaxKind.IdentifierToken);
-        }
+        public bool IsIdentifier(SyntaxToken token) => token.IsKind(SyntaxKind.IdentifierToken);
 
-        public bool IsGlobalNamespaceKeyword(SyntaxToken token)
-        {
-            return token.IsKind(SyntaxKind.GlobalKeyword);
-        }
+        // In C# parameter's identifier can only be Token, never Node
+        public bool IsParametersIdentifier(SyntaxNode node) => false;
```

Comment:
this is subtle and potentially confusing for callers.  I would not have this, and would likely just have the core refactoring service have a VB specific path in the subclass.

Reply:
Will get fix through lifting it to RefactoringHelpersService